<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewComputeWin</class>
 <widget class="QDialog" name="NewComputeWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Computation</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="rows">
        <property name="text">
         <string>   Rows(5-8):   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rowLine">
        <property name="text">
         <string>5</string>
        </property>
        <property name="maxLength">
         <number>32766</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="inp1Label">
        <property name="text">
         <string>     Input #1:     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inp1">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="inp2Label">
        <property name="text">
         <string>     Input #2:     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inp2">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="out1Label">
        <property name="text">
         <string>    Output #1:   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="out1">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="out2Label">
        <property name="text">
         <string>    Output #2:   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="out2">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="out3Label">
        <property name="text">
         <string>    Output #3:   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="out3">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
